.travis.yml

```diff
@@ -26,6 +26,8 @@ env:
   - TESTS="1" ODOO_REPO="OCA/OCB"
   - TESTS="1" ODOO_REPO="odoo/odoo" INCLUDE="website_signup_legal_page_required"
   - TESTS="1" ODOO_REPO="OCA/OCB" INCLUDE="website_signup_legal_page_required"
+  - TESTS="1" ODOO_REPO="odoo/odoo" EXCLUDE="website_seo_redirection"
```

Comment:
I'm pretty sure this time is different. I think if we are adding a module that modifies all of Odoo's behavior, obviously the cost of adding this line is that there's another build to try, but the benefit is that in case of failure, the :x: will be absolutely visible by anyone and will effectively save the contributor a big amount of time, not only in seeing where's the bug, but also in digging through the `.travis.yml` file to find a comment that somebody left there assuming people reads that file before pushing their PR.

I think this module is somehow like the whole OCB repo: it is supposed to be 100% compatible with existing installations, but the only way to ensure that is to test it on every build.

But I still can add a comment to it to explain why it's necessary.